views/pages/index.js

```diff
@@ -5,11 +5,16 @@ module.exports = function(options) {
 
   str += '<h1>Open Fixture Library</h1>';
 
-  str += '<p>To use lighting control software like <a href="http://www.qlcplus.org/">QLC+</a> or <a href="http://www.ecue.de/">e:cue</a>, you need fixture definition files for your specific hardware. Those can be difficult to create or find for your software or they may be wrong because nobody peer-reviewed them.</p>';
+  str += '<p>To use lighting control software like <a href="http://www.qlcplus.org/">QLC+</a> or <a href="http://www.ecue.de/">e:cue</a>, you need proprietary fixture definition files that describe your lighting hardware. Those can be difficult to create, find or convert from one format into another.</p>';
 
-  str += '<p><abbr title="Open Fixture Library">OFL</abbr> tries to solve those problems by collecting fixture definitions and making them downloadable in various formats. Everybody can <a href="https://github.com/FloEdelmann/open-fixture-library">contribute</a> and help to improve!</p>';
+  str += '<p><abbr title="Open Fixture Library">OFL</abbr> tries to solve those problems by collecting fixture definitions and making them downloadable in various formats. Everybody can <a href="https://github.com/FloEdelmann/open-fixture-library">contribute</a> and help to improve! Thanks!</p>';
 
-  str += '<p>The project is still in a very early state, but you can still check out the progress on <a href="https://github.com/FloEdelmann/open-fixture-library">GitHub</a> and also already report issues or feature requests or even contribute some code. Thanks!</p>';
+  str += '<div class="banner grid">';
+  str += '<a href="/manufacturers/" class="card">' + require('../includes/svg')({svgBasename: 'folder-multiple'}) + '<h2>Browse fixtures</h2></a>';
+  str += '<a href="https://github.com/FloEdelmann/open-fixture-library/issues?q=is%3Aopen+is%3Aissue+label%3Abug" class="card">' + require('../includes/svg')({svgBasename: 'bug'}) + '<h2>Report problem</h2></a>';
+  str += '<a href="https://github.com/FloEdelmann/open-fixture-library/issues?q=is%3Aopen+is%3Aissue+-label%3Abug" class="card">' + require('../includes/svg')({svgBasename: 'lightbulb-on-outline'}) + '<h2>Request feature</h2></a>';
```

Comment:
It's a good idea having a link to all bug issues and all non-bug issues, but the bug label is called `type-bug` and not `bug`.